bin/git-coauthor

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+coauthor=$1
+email=$2
+
+test -z $coauthor && echo "co-author required." 1>&2 && exit 1
```

Comment:
We should quote the "$coauthor". Otherwise we will be in trouble when the input contains whitespace. Like this one: `git coauthor "John Doe" john@doe`.